gradle/timelock.gradle

```diff
@@ -0,0 +1,35 @@
+dependencyRecommendations {
```

Comment:
Awesome! It's about time we started combining these together, so thanks for doing this 😄